portfolio/src/main/java/com/google/sps/servlets/VideoServlet.java

```diff
@@ -0,0 +1,41 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.Random; 
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that helps avoid getting the same video twice in a row.*/
+@WebServlet("/video")
+public class VideoServlet extends HttpServlet {
+
+  public int lastVideoIndex = 4;
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    Random randomGenerator = new Random();
+
+    // Generate a random index from 0 to 5.
+    int nextVideoIndex = randomGenerator.nextInt(5);
```

Comment:
Consider having another variable that determines the length of videos available, so that you don't have to hardcode the value 5. This could either be done dynamically by accessing the list of videos you have, or you could define a constant variable so that it's easier to modify later on. You could imagine a scenario where you delete one of the videos or decide to add 10 more videos - this reduces the amount of code you would have to update.

Reply:
Done!

Reply:
Done!